fonts.css

```diff
@@ -1,27 +1,31 @@
 @font-face {
-  font-family: 'ptserif';
-  src: url('ptserif-bold-webfont.woff') format('woff');
+  font-family: 'PTSerif';
+  src: url('fonts/PTSerifBold.woff2') format('woff2'), 
+  url('fonts/PTSerifBold.woff') format('woff');
   font-weight: bold;
   font-style: normal;
 }
 
 @font-face {
-  font-family: 'ptserif';
-  src: url('ptserif-bolditalic-webfont.woff') format('woff');
+  font-family: 'PTSerif';
+  src: url('fonts/PTSerifBoldItalic.woff2') format('woff2'),
+  url('fonts/PTSerifBoldItalic.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)